### PR TITLE
TSDK-249 Add support to broker for DAMLhub hosted app

### PR DESCRIPTION
**Pull request by mundacho**
## Purpose

The purpose of this PR is to adapt the demo application to support DAML hub.

## Approach

DAML Hub requires triggers to be in a separate project. Thus we separated the js-daml-demo into two projects: js-daml-pollnft-app and daml-pollnft-trigger. The daml-pollnft-trigger only contains the triggers.

## Testing

The project was successfully deployed to DAMLHub.

## Tickets
* Topl/Brambl#136

### CODE REVIEW

File: daml-pollnft-trigger/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+During all of these interactions, the user interacts with DAML contracts, a broker intercepts the calls to perform
+operations on the blockchain and triggers handle the contracts created by the broker.
+
+## Requirements
+
+This demo was tested under Windows using Windows Subsystem for Linux (WSL) and ubuntu. The following software
+is required:
+
+- [Daml]: https://docs.daml.com
+- [Node.js]: https://nodejs.dev
```
Comment (DiademShoukralla) [inline daml-pollnft-trigger/README.md:21]: These two lines do not show up in the rendered markdown
image

File: daml-pollnft-trigger/README.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+npm run-script build
+zip -r ../js-daml-app-ui.zip build
+```
+
+Next you need to create a ledger on [Daml Hub], upload the files
+`.daml/dist/js-daml-app-0.1.0.dar` (created by the `daml build` command)
+and `js-daml-app-ui.zip` (created by the `zip` command based on the result
+of `npm run-script build`).
+
+[Daml Hub]: https://hub.daml.com
```
Comment (DiademShoukralla) [inline daml-pollnft-trigger/README.md:103]: This line is also not rendered

File: daml-pollnft-trigger/README.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+operations on the blockchain and triggers handle the contracts created by the broker.
+
+## Requirements
+
+This demo was tested under Windows using Windows Subsystem for Linux (WSL) and ubuntu. The following software
+is required:
+
+- [Daml]: https://docs.daml.com
+- [Node.js]: https://nodejs.dev
+- The broker software (available [here](https://github.com/Topl/daml-demos/tree/main/scala-daml-broker-app)).
```
Comment (DiademShoukralla) [inline daml-pollnft-trigger/README.md:22]: I am unable to access the link
image

**Review by hwbehrens — APPROVED** [review]
Looks good!

**Review by DiademShoukralla — APPROVED** [review]
Looks good! I just noted a few minor issues in the rendered README.md

File: daml-pollnft-trigger/README.md (hunk 1 above)
Comment (mundacho) [reply daml-pollnft-trigger/README.md:21]: fixed

File: daml-pollnft-trigger/README.md (hunk 3 above)
Comment (mundacho) [reply daml-pollnft-trigger/README.md:22]: fixed

File: daml-pollnft-trigger/README.md (hunk 2 above)
Comment (mundacho) [reply daml-pollnft-trigger/README.md:103]: fixed

**Pull request merged**